.github/workflows/auto-update-Dockerfiles.yml

```diff
@@ -0,0 +1,195 @@
+name: Auto-Update Lambda Dockerfiles Monthly
+
+permissions:
+  contents: write
+  pull-requests: write
+
+on:
+  # Run monthly on the 1st day of each month at midnight UTC
+  schedule:
+    - cron: '0 0 1 * *'
```

Comment:
we should run this daily. We don't want to be a month late for the patches. Our goal is to get the updated OCIs to Lambda as quickly as possible.